main.py

```diff
@@ -471,6 +532,9 @@ def run(prompt_queue, address='', port=8188):
     except:
         pass
 
-    run(q, address=address, port=port)
+    try:
+        loop.run_until_complete(run(q, socket_handler, address=address, port=port))
+    except KeyboardInterrupt:
```

Comment:
My ctrl-c issue is fixed if I remove this try/except.

Reply:
I've removed it